test/screenshot.test.js

```diff
@@ -80,41 +45,14 @@ function fillImageToSize(image, width, height) {
 const skip = !!process.env.SKIP_VISUAL_REGRESSION_TEST;
 
 describe('screenshot visual regression', { skip, concurrency: true }, () => {
-  /** @type {import('esbuild').BuildContext} */
-  let esbuildContext;
-
-  /** @type {number} */
-  let port;
-
-  /** @type {import('puppeteer').Browser} */
-  let browser;
-
-  /** @type {import('puppeteer').BrowserContext} */
-  let browserContext;
-
-  before(async () => {
-    esbuildContext = await esbuild.context({});
-    port = (await esbuildContext.serve({ servedir: 'dist' })).port;
-    browser = await puppeteer.launch({ headless: 'new' });
-    browserContext = await browser.createIncognitoBrowserContext();
-  });
-
-  after(async () => {
-    await Promise.all([browserContext.close(), browser.close(), esbuildContext.dispose()]);
-  });
-
   it('has pages to test', () => {
     assert(paths.length);
   });
 
   paths.forEach((path) => {
     test(path, async () => {
-      const localURL = `http://localhost:${port}`;
-      const page = await browserContext.newPage();
-      const local = await getScreenshot(page, localURL + path);
-      const remote = await getScreenshot(page, REMOTE_HOST + path);
-      const localPNG = PNG.sync.read(local);
-      const remotePNG = PNG.sync.read(remote);
+      const localPNG = PNG.sync.read(await readFile(join(BRANCH_SNAPSHOTS_DIRECTORY, path)));
+      const remotePNG = PNG.sync.read(await readFile(join(MAIN_SNAPSHOTS_DIRECTORY, path)));
```

Comment:
now that now that these are both "local" should we call it "branch" and "main"?
```suggestion
      const branchPNG = PNG.sync.read(await readFile(join(BRANCH_SNAPSHOTS_DIRECTORY, path)));
      const mainPNG = PNG.sync.read(await readFile(join(MAIN_SNAPSHOTS_DIRECTORY, path)));
```

Reply:
Yep, good call, I think I'd meant to circle back to that but forgot.

Reply:
Updated in e7b889d.